guides/decrypt-api-key-in-action/src/index.ts

```diff
@@ -0,0 +1,54 @@
+import { LitNodeClient, encryptString } from "@lit-protocol/lit-node-client";
+import { genWallet } from "./wallet";
+import { genSession } from "./session";
+import { genActionSource} from "./contstants";
+
+const url = `<your http endpoint for api-key usage>`;
+const key = '<your api key>';
+
+let client = new LitNodeClient({
+    litNetwork: 'cayenne'
+});
+await client.connect();
+
+const wallet = genWallet();
+
+const session = await genSession(wallet, client, []);
```

Comment:
Resources don't include the ability to execute a Lit Action. Does this get added by default?

Reply:
no but we do not need any when encrypting, I just updated to include them when we are calling the lit action